modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -105,6 +105,16 @@ All page names are validated:
 * Page names cannot contain these characters: `@` `*` `+` `/` `_` `\`
 * Page names cannot start with `.`
 
+The special page *Home* has the URL `pages/home` and the special page *Terms and conditions* has the URL `pages/.terms`.
+
+Exchange sets a URL for other new pages that contains a page ID of three alphanumeric characters, a hyphen, and three more alphanumeric characters, such as `pages/123-abc/Formats`. Changing the page name from *Formats* to *Formatting* changes the URL to `pages/123-abc/Formatting`, and the old URL automatically redirects to the new URL.
+
+Some older asset pages have URLs that use the previous format with no page ID, such as `pages/Messages`. Changing the page name from *Messages* to *Messaging* changes the URL to a URL with the new format such as `pages/def-456/Messaging`, and the old URL automatically redirects to the new URL.
```

Comment:
Here `Some older asset pages` is a bit vague, is it possible to state something like `Pages created before <release date>` or do we want to avoid being that specific? What do you think, @wwarner-salesforce ?

Reply:
Good point! The page "Describe an Asset" states release dates. I like your idea to do that here too. I made that change. Would you please review and approve the PR?